https://bugs.gentoo.org/909738
https://github.com/numpy/numpy/pull/25078

commit 43aaf2093d8dfb3c1fea5d409ea4aa1d0f77816f
Author: matoro <matoro@users.noreply.github.com>
Date:   Mon Nov 6 10:21:32 2023 -0500

    BUG: alpha doesn't use REAL(10)
    
    Same as e.g. loongarch per gh-24904.  At this point seems like it should
    be more of an exclude list than an include one...

diff --git a/numpy/f2py/crackfortran.py b/numpy/f2py/crackfortran.py
index f352bbaa2..d17b052f9 100755
--- a/numpy/f2py/crackfortran.py
+++ b/numpy/f2py/crackfortran.py
@@ -2452,7 +2452,7 @@ def _selected_real_kind_func(p, r=0, radix=0):
     if p < 16:
         return 8
     machine = platform.machine().lower()
-    if machine.startswith(('aarch64', 'arm64', 'loongarch', 'power', 'ppc', 'riscv', 's390x', 'sparc')):
+    if machine.startswith(('aarch64', 'alpha', 'arm64', 'loongarch', 'power', 'ppc', 'riscv', 's390x', 'sparc')):
         if p <= 33:
             return 16
     else:
